views/post-page.ejs

```diff
@@ -41,25 +41,9 @@
         <style id="cssHolder"></style>
 
         <link href="https://fonts.googleapis.com/css?family=IBM+Plex+Sans:300,400,500,700&display=swap" rel="stylesheet" />
-
-        <script async="" src="https://www.google-analytics.com/analytics.js"></script>
-        <script>
-            (function (i, s, o, g, r, a, m) {
-                i["GoogleAnalyticsObject"] = r;
-                (i[r] =
-                    i[r] ||
-                    function () {
-                        (i[r].q = i[r].q || []).push(arguments);
-                    }),
-                    (i[r].l = 1 * new Date());
-                (a = s.createElement(o)), (m = s.getElementsByTagName(o)[0]);
-                a.async = 1;
-                a.src = g;
-                m.parentNode.insertBefore(a, m);
-            })(window, document, "script", "https://www.google-analytics.com/analytics.js", "ga");
-            ga("create", "UA-25629695-1", "auto");
-            ga("send", "pageview");
-        </script>
```

Comment:
why are you removing Google Analytics?

Reply:
We meant to remove it long ago from the blog, most of the scripts were removed already. Patrick has more details as to why.